docs/reference/remap/length.cue

```diff
@@ -0,0 +1,63 @@
+package metadata
+
+remap: functions: length: {
+	arguments: [
+		{
+			name:        "value"
+			description: "The array or map"
+			required:    true
+			type: ["array", "map", "string"]
+		},
+	]
+	internal_failure_reason: null
+	return: ["integer"]
+	category: "Enumerate"
+	description: """
+		Returns the length of the input:
+
+		* If an array, the size of the array
+		* If a string, the length of the string
```

Comment:
```suggestion
		* If a string, the number of bytes in the string
```

I think otherwise this is ambiguous as to whether it means bytes or characters.